lib/jsdom/level1/core.js

```diff
@@ -245,7 +245,7 @@ core.DOMImplementation.prototype = {
 };
 
 core.Node = function Node(ownerDocument) {
-  EventTarget.call(this);
+  EventTarget.init.call(this);
```

Comment:
I mean, window is a special case. I think we fixed symbols in new io.js, but there are probably issues there in general, yeah.

Still not entirely sure what the problem is though. All the symbols stay on the impl instance. Note that it's `this[impl].method.apply(this[impl], args)`, not `this[impl].method.apply(this, args)`.